tests/common/config_reload.py

```diff
@@ -105,6 +106,13 @@ def config_reload_minigraph_with_rendered_golden_config_override(
                   override_config=True, golden_config_path=golden_config_path, is_dut=is_dut)
 
 
+def pfcwd_feature_enabled(duthost):
+    localhost_config = duthost.shell('sonic-db-dump -n CONFIG_DB -y -k \"DEVICE_METADATA|localhost\"')
+    meta_data = json.loads(localhost_config["stdout"])
```

Comment:
For consistency purposes, it might be better to use `duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']['DEVICE_METADATA']['localhost']`. (This also protects against an old image running where `sonic-db-dump` isn't available.

Reply:
Fixed